packages/react/src/components/MultiSelect/FilterableMultiSelect.tsx

```diff
@@ -854,6 +898,8 @@ const FilterableMultiSelect = React.forwardRef(function FilterableMultiSelect<
             className={inputClasses}
             {...inputProps}
             ref={mergeRefs(textInput, inputProps.ref)}
```

Comment:
If its similar to const `mergedRef`, can we use that here?

Reply:
done